docs/README.rst

```diff
@@ -368,6 +386,8 @@ sslProfile [#ssl]_   JSON object       Optional                   BIG-IP SSL pro
 
    You can also `assign IP addresses to BIG-IP virtual servers using IPAM`_.
 
+   See `Source Address Translation Overview`_ for more information on configuring virtual server source address translation.
```

Comment:
Change to "See the `Source Address Translation Overview`_ on AskF5 ..."

Reply:
Changed.